src/Apps/Order/Routes/Shipping2/__tests__/Shipping2.jest.tsx

```diff
@@ -345,7 +345,7 @@ describe("Shipping", () => {
           relayEnv
         )
 
-        expect(screen.getAllByRole("button", { name: "Offer" }).length).toBe(2)
+        expect(screen.getAllByRole("button", { name: "Offer" }).length).toBe(1)
```

Comment:
Suggestion(non-blocking): maybe we could check for visibility instead?

```suggestion
        expect(screen.getByRole("button", { name: "Offer" })).toBeVisible()
```

Reply:
The test failed when I asserted its visibility, so I took the shortcut of counting buttons instead. I agree that asserting its visibility is the better way to write this test.